control/prometheus.py

```diff
@@ -0,0 +1,196 @@
+#
+#  Copyright (c) 2021 International Business Machines
+#  All rights reserved.
+#
+#  SPDX-License-Identifier: LGPL-3.0-or-later
+#
+#  Authors: pcuzner@ibm.com
+#
+
+import os
+import logging
+import spdk.rpc as rpc
+from collections import namedtuple
+from prometheus_client.core import REGISTRY, GaugeMetricFamily, CounterMetricFamily, InfoMetricFamily
+from prometheus_client import start_http_server
+
+
+RBD = namedtuple("RBD_Lookup", "pool image")
```

Comment:
Not a big deal but the `typing` library offers a much friendlier and typed data structure:
```py
class RBD(NamedTuple):
   pool: str
   image: str
```
BTW, at some point this might also require specifying the RBD Namespace: @idryomov ?

Reply:
Yes.  `create_bdev` RPC still doesn't support RBD namespaces, but it's going to happen at some point.  Images would be identified by `(pool, namespace, image)` tuples instead of just `(pool, image)`.

Reply:
Thanks @idryomov @epuertat. I'll update.